css/styles.css

```diff
@@ -541,3 +541,61 @@ main {
         font-size: 1.5rem;
         color: #596780;
         font-weight: 700; }
+
+#pomodoro-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center; }
+  #pomodoro-container .pomodoro__heading {
+    text-align: center;
+    color: #0D121F;
+    font-size: 2.4rem;
+    font-weight: 700; }
+  #pomodoro-container .notification {
+    min-width: 320px;
+    max-width: 320px;
+    background-color: #E7DEFE;
+    color: #7C5CFC;
+    font-weight: 700;
+    font-size: 1.5rem;
+    padding: 1rem;
+    margin-top: 3rem;
+    text-align: center;
+    border-radius: 0.5rem; }
+  #pomodoro-container .pomodoro__body {
```

Comment:
This looks so congested without 0 lines. Can you see how to auto-format this?

Reply:
It has been changed in the latest PRs.

Reply:
Update: styles.css has to be formatted. It was styles.scss that had been formatted in the latest PR. 
https://github.com/jazzcodes/ProductivityZap/pull/13/commits/7060e9573aab0281ade29ee04e0ce834a63053ed

Will make the required changes in the styles.css file.